spec/lib/miq_expression_spec.rb

```diff
@@ -2216,11 +2216,6 @@
       expect(subject).to eq(:datetime)
     end
 
-    it "with managed-field" do
-      @field = "managed.location"
```

Comment:
it was fixed https://github.com/ManageIQ/manageiq/pull/13769/files#diff-85af62b23d0d2367151a498d0cf7737aR5